src/sharedHelpers/clearCaches.ts

```diff
@@ -53,13 +53,13 @@ const clearSyncedMediaForUpload = async realm => {
     unsyncedPhotoFileNames
   );
 
-  // Clean out photos
+  // Clean out sounds
   const unsyncedObservationsWithSounds: RealmObservation[] = realm
     .objects( "Observation" )
     .filtered( "observationSounds._synced_at == nil" );
   const unsyncedSoundFileNames = unsyncedObservationsWithSounds
     .map( observation => observation.observationSounds.map(
-      os => os.sound.fileUrl?.split( "soundUploads/" )?.at( 1 )
+      os => os.sound.file_url?.split( "soundUploads/" )?.at( 1 )
```

Comment:
Would it be a good idea to also modify line 19 in this file? The interface there no longer matches this property name.

Reply:
good catch

Reply:
Thanks @angielt! Looks good to me now.